docs/about-us.rst

```diff
@@ -0,0 +1,61 @@
+.. SPDX-FileCopyrightText: 2023 The meson-python developers
+..
+.. SPDX-License-Identifier: MIT
+
+.. _about-us:
+
+********
+About Us
+********
+
+``meson-python`` is a project that extends the Meson_ build system, making it
+possible to it for Python_ packaging. It is 100% open source and is available
+under the permissive `MIT (Expat) license`_. The project was started in 2021,
+and moved under the umbrella of `The Meson Build System organization`_ in 2022.
+
+
+Contributors
+============
+
+We are a community-driven project, so please check out the `complete list of all
+contributors`_ for a list of everyone that helped shape ``meson-python``.
+
+Another big part of the project are the maintainers, which guide the
+development:
+
+- :user:`Daniele Nicolodi (dnicolodi) <dnicolodi>`
+- :user:`Filipe Laíns (FFY00) <FFY00>`
+- :user:`Henry Schreiner (henryiii) <henryiii>`
+- :user:`Ralf Gommers (rgommers) <rgommers>`
+- :user:`Thomas Li (lithomas1) <lithomas1>`
+
+Finally, since we build on top of Meson_, this project is only possible thanks
+to all the Meson_ contributors_ and maintainers.
+
+
+Institutional Partners
```

Comment:
This was pretty much copied from NumPy's documentation. I really don't want to argue about it, so I'll defer to @rgommers.

Reply:
I think you refer to the "Institutional Partners" section on this document https://numpy.org/about/. This needs to be read paired with this https://numpy.org/doc/stable/dev/governance/governance.html#institutional-partners-and-funding Having to define the "Instituttional Partner" role to state that two of the maintainers of meson-python do that as part of their job seems useless indirection to me. I like to keep things simple.

Reply:
Yes, the "Institutional Partners" thing is a specific term that is defined in the governance docs of not only NumPy, but Jupyter (where it originated), SciPy, and some other large projects. We don't have a meaning for this term in `meson-python` and I think the project is way too small to deal with those kinds of formalities. 

So, I'm fine with using "Funding". 

> I'm not even sure we need the company logo.

I don't really mind either way. It has been a substantial investment, so I'd say showing a logo is justified. But it's not something I asked for so we can leave it out for the time being. 

In case the funding section grows we may need some criteria about when to list something, but let's cross that bridge if and when we get to it.

Reply:
I'm not at all opposed to having the logo. It was not clear to me the reason why the affiliation of the maintainers is reported. If it is to give back a bit in term of visibility I think having the logo makes sense.

Reply:
Great, thanks. In that case I'd prefer keeping it.